src/adminactions/merge.py

```diff
@@ -73,7 +73,7 @@ class Media:
         js = [
             'admin/js/vendor/jquery/jquery.js',
             'admin/js/jquery.init.js',
-            'adminactions/js/merge.min.js',
```

Comment:
What's the reason to no longer use the minified version?

Reply:
Tested this with the minified vs the plain, and the minified had the original bug in it while the plain worked as expected. 
Good call though, I'll put in a bit of effort to see if I can get the minified version working; no use having a phantom bug waiting for us down the line.

Reply:
Hmm, yeah just pushed tried to minify it myself ([UglifyJS 3](https://skalman.github.io/UglifyJS-online/)) and the minified version doesn't work :(
Might be worth it to disable some minification options and try a few more times.

Reply:
Would you min removing the minified version then? Somebody (most likely me) will come along shortly and try to switch to it.